aws_iam_policies/aws_access_keys_at_account_creation.py

```diff
@@ -20,7 +19,10 @@ def policy(resource):
         return True
 
     create = resource["TimeCreated"]
-    key_rot_date = datetime.datetime.strptime(key_rot, AWS_TIME_FORMAT)
-    create_date = datetime.datetime.strptime(create, PANTHER_TIME_FORMAT)
+    key_rot_date = datetime.strptime(key_rot, AWS_TIME_FORMAT)
+    try:
+        create_date = datetime.strptime(create, PANTHER_TIME_FORMAT)
+    except ValueError:
+        create_date = datetime.strptime(create, AWS_TIME_FORMAT)
```

Comment:
could it also raise an exception in the except clause if it doesn't match AWS_TIME_FORMAT? I don't know if this is any better, but another option could be: 
```suggestion
        for format in [PANTHER_TIME_FORMAT, AWS_TIME_FORMAT]:
            try:
                create_date = datetime.strptime(create, format)
                break
            except ValueError:
                continue
```

🤷🏻‍♀️

Reply:
I like this form better -- let's revisit it if timestamp issues become more prevalent.